### PR TITLE
preparation for docs automation

### PR DESCRIPTION
**Pull request by jkremser**
This PR prepares the ground in gh-pages branch for the docs auto-generation.

### CODE REVIEW

**Comment by cfcosta** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -1,311 +0,0 @@
-<!DOCTYPE html>
```
Comment (josejulio) [inline index.html:1]: ~~Does github follow symlinks?~~
Never mind, i confused it with the README .

**Comment by josejulio** [comment]
~~It confuses me that these files are marked `File renamed without changes.` but i don't see them anywhere in the code.
I suppose that these are autogenerated, should we include them in the repo?~~

After talking with my rubber ducky he correctly pointed me out that this PR target is `gh-pages` and not `master`.

**Comment by israel-hdez** [comment]
@josejulio You can find the renamed files on gh-pages branch. That branch is not a branch of master, it is the branch containing only documentation... 
That's weird...

**Comment by jkremser** [comment]
Hmm, there is an issue with the `.travis.docs.rb` script being called four times in that `after_success:` section. Because we have 4 permutations of builds. It would be handy if travis provides a feature like this https://github.com/travis-ci/travis-ci/issues/929 I am currently looking into this guy (https://github.com/dmakhno/travis_after_all) if it can help.

EDIT: I solved it by https://github.com/hawkular/hawkular-client-ruby/pull/198/files#diff-515e88bb27e03f935a08a01e14f1d448R17

**Review by israel-hdez — APPROVED** [review]
LGTM

**Pull request merged**